### ISSUE TITLE
Cassowary breaks 'arc linters' command

### ISSUE THREAD

**Issue by paulstringer** [imobile3/cassowary#6]
With Cassowary loaded, running arc linters generates the following error

```ERROR 2: Missing argument 1 for ArcanistAndroidLinter::__construct() at [/Volumes/Work/cassowary/libcassowary/src/lint/linter/ArcanistAndroidLinter.php:42]
  #0 ArcanistAndroidLinter::__construct()
  #1 ReflectionClass::newInstance() called at [<phutil>/src/utils/utils.php:664]
  #2 newv(string, array) called at [<phutil>/src/symbols/PhutilSymbolLoader.php:313]
  #3 PhutilSymbolLoader::loadObjects() called at [<arcanist>/src/workflow/ArcanistLintersWorkflow.php:41]
  #4 ArcanistLintersWorkflow::run() called at [<arcanist>/scripts/arcanist.php:338]```

**Comment by mbishopim3** [comment]
Confirmed. We'll investigate, probably a breaking change the Phab devs made.

**Comment by mbishopim3** [comment]
Resolved by e74aea190d338e52f82754d232756411568f8cd0.
